packages/kit/src/core/create_manifest_data/index.js

```diff
@@ -140,18 +140,24 @@ export default function create_manifest_data({ config, output, cwd = process.cwd
 			params.push(...item.parts.filter((p) => p.dynamic).map((p) => p.content));
 
 			if (item.is_dir) {
+				const layout_reset = find_layout('$layout.reset', item.file);
 				const layout = find_layout('$layout', item.file);
 				const error = find_layout('$error', item.file);
 
+				if (layout_reset && layout) {
+					throw new Error(`Cannot have $layout next to $layout.reset: ${layout_reset}`);
```

Comment:
Why is this necessary? I think many people may find this rule surprising; I would expect that if `$layout.reset.svelte` and `$layout.svelte` are both present in the `admin` directory, the rule is "do not inherit any parent layouts, and start with *this* `$layout.svelte` as the top-level layout for this directory and any subdirectories." With the "Cannot have $layout next to $layout.reset" rule in place, having a special admin layout for `/admin/foo` and `/admin/bar` would require `/src/routes/admin/$layout.reset.svelte` and then `/src/routes/admin/foo/index.svelte` and `/src/routes/admin/bar/index.svelte`, with a duplicated `$layout.svelte` in both `/src/routes/admin/foo` and `/src/routes/admin/bar`. Whereas if the rule was "$layout.reset applies first, then any $layout in the same folder applies after the reset" then you could have a `/src/routes/admin` directory containing four files: `$layout.reset.svelte` to stop inheritance, `$layout.svelte` to set a layout for all admin pages, and `foo.svelte` and `bar.svelte` to be separate pages. Much simpler, and IMHO better, directory structure.

Reply:
Oh wait, I may have misunderstood. The `$layout.reset.svelte` is itself the layout for its directory and below. Never mind. I thought `$layout.reset.svelte` was an empty file, hence my confusion. Ignore the above, sorry.